weedcoco/validation.py

```diff
@@ -191,6 +191,13 @@ def validate(weedcoco, images_root=None, schema="weedcoco"):
         validate_image_sizes(weedcoco, images_root)
 
 
+def fix_compatibility_quirks(weedcoco):
```

Comment:
Not sure this belongs in `weedcoco.validation`, mostly because this module has a single entry point (and should not modify the content ordinarily).

Reply:
I was also unsure whether it belonged there. I think you're right, I've moved it to `weecoco.utils`.